### DISCUSSION TITLE
Darker fails with Type Error, exit code 1 after MacOS update

### DISCUSSION THREAD

**Discussion by zachnorton4C** [akaihola/darker#396] [Q&A]
**Describe the bug**
After upgrading to MacOS Monterey 12.5.1, I am unable to get darker to run on pre-commit -- it fails with a TypeError.

**To Reproduce**
1. Attempt to create a new commit
2. Darker exits with code: 1
3. See traceback error: 

```python
Traceback (most recent call last):
  File "bin/darker", line 8, in <module>
    sys.exit(main())
  File "darker/__main__.py", line 196, in main
    args, config, config_nondefault = parse_command_line(argv)
  File "darker/command_line.py", line 163, in parse_command_line
    config = load_config(args.src)
  File "darker/config.py", line 37, in load_config
    path = find_project_root(tuple(srcs or ["."])) / "pyproject.toml"
TypeError: unsupported operand type(s) for /: 'tuple' and 'str'
```

**Expected behavior**
Until recent MasOS update, darker was running without issue.

**Environment:**
 - OS: MacOS Monterey 12.5.1
 - Python version 3.8.9
 - Git version 2.37.3
 - Darker version 1.5.0
 - Black version 22.8.0
 - other reformatter and linter versions isort==5.10.1

**Additional context**
I attempted with several earlier versions of black and darker and saw the same result.

**Comment by akaihola** [comment]
>```python
>  File "darker/config.py", line 37, in load_config
>    path = find_project_root(tuple(srcs or ["."])) / "pyproject.toml"
>```

>- Darker version 1.5.0

Hi @zachnorton4C, thanks for the issue report!

You don't seem to be running Darker 1.5.0 in the pre-commit environment, but version 1.2.x instead, since versions 1.2.0–1.2.4 are the only ones which had the `find_project_root()` call on line `darker/config.py:37`.

How did you initialize pre-commit? What does your `.pre-commit-config.yaml` look like?

**Comment by zachnorton4C** [comment]
Hi @akaihola Thank you for the follow up! Your question enlightened another path for me and my team and I were able to resolve this. Thanks!
